intro-Linux/ch_getting_started.tex

```diff
@@ -3,7 +3,7 @@ \chapter{Getting Started}
 \section{Logging in}
 
 To get started with the HPC-UGent infrastructure, you need to obtain a VSC
-account, see \url{http://hpc.ugent.be/userwiki/index.php/User:VscRequests}.
+account, see \url{http://hpcugent.github.io/vsc_user_docs/pdf/intro-HPC-linux-gent.pdf#chapter.2}.
```

Comment:
Maybe make this a bit more "user friendly", like this?

```
\href{<url>}{Chapter 2 of the VSC user manual}
```

Do check whether that renders as expected though, i.e. whether it's clear that's a link to another document or not...